app/controllers/attachment_references_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AttachmentReferencesController < ApplicationController
+  load_resource :attachment_reference
+  before_action :authorize_resource
+
+  def show
+    redirect_to @attachment_reference.url
+  end
+
+  private
+
+  def authorize_resource
+    authorize!(:read, @attachment_reference.attachable)
```

Comment:
so that users cannot manage to get all the attachments by increasing the id in `attachments/:id`. That is similar to our trackable jobs ?